auth-server/README.md

```diff
@@ -29,6 +29,7 @@ To deploy the [`Auth server`](/auth-server/) manually:
 1. [Create](https://cloud.google.com/compute/docs/instances/create-start-instance) a Compute Engine VM instance with your preferred machine type and OS (for example, e2-medium and Debian 10), then [reserve a static IP](https://cloud.google.com/compute/docs/ip-addresses/reserve-static-internal-ip-address)
 1. Verify that your VM instance has the `Stackdriver Logging API` write [access scope](https://cloud.google.com/compute/docs/access/service-accounts#accesscopesiam) (on by default) and that your VM’s [service account](https://cloud.google.com/compute/docs/access/service-accounts#default_service_account) has the [`Logs Writer`](https://cloud.google.com/logging/docs/access-control) role (off by default) 
 1. Check out the latest code from the [FDA MyStudies repository](https://github.com/GoogleCloudPlatform/fda-mystudies/)
+1. Update the landing page logo as desired replacing the image [landingpage-logo.png](oauth-scim-service/src/main/resources/static/images/landingpage-logo.png) (keep the same file name)
```

Comment:
Is there a resolution requirement for the image?

Reply:
Also, let's rephrase to "Optional. If desired, update landing page icon by replacing the image [`landingpage-logo.png`](oauth-scim-service/src/main/resources/static/images/landingpage-logo.png) with a new image of resolution XYZ (keep the filename the same)"

Reply:
Add the `` to the filename like in the comment above

Reply:
Updated [README.md](https://github.com/GoogleCloudPlatform/fda-mystudies/blob/2.0.2-release-candidate/auth-server/README.md) of Auth server

Reply:
FYI I went and fixed this myself